swift/extractor/translators/StmtTranslator.cpp

```diff
@@ -73,7 +73,7 @@ codeql::ForEachStmt StmtTranslator::translateForEachStmt(const swift::ForEachStm
   auto entry = dispatcher.createEntry(stmt);
   fillLabeledStmt(stmt, entry);
   entry.body = dispatcher.fetchLabel(stmt.getBody());
- // entry.sequence = dispatcher.fetchLabel(stmt.getTypeCheckedSequence());
+  // entry.sequence = dispatcher.fetchLabel(stmt.getTypeCheckedSequence());
```

Comment:
Why not just delete this line?

Reply:
this was autofromatted by by my IDE, I have no idea what it is :sweat_smile:

Reply:
yeah, I removed it, it was an unneeded leftover :+1: